packages/template-retail-react-app/app/translations/README.md

```diff
@@ -91,24 +97,31 @@ Here are a few useful things to know for developers.
 
 How a locale gets chosen depends on whether there’s a match found between 2 sets of locales. On a high level, it looks like this:
 
-1. Get the app-supported locales, which are defined in `pwa/package.json` (under `l10n.supportedLocales` ).
+1. Get the app-supported locales, which are defined in each site object in `config/sites.js` (under `l10n.supportedLocales` of your targeted site).
 2. Get the user-preferred locales, which are what the visitors prefer to see. The developer is responsible for fully implementing them in their own projects within the special `_app` component.
 3. If there’s a match between these 2 sets of locales, then the app would use it as the target locale.
-4. Otherwise, the app would fall back to its default locale (which is defined in `package.json` under `l10n.defaultLocale` ).
+4. Otherwise, the app would fall back to the locale of the inline `defaultMessage`s. 
 
 ### How to Detect the Active Locale
 
--   Within component render, `useIntl` hook is available to you: `const {locale} = useIntl()`
--   Within a page’s `getProps` there’s a `params` prop (specifically `params.locale`) coming from the routes that can tell you about which locale is being used in the current page URL.
+- Within component render, `useLocale` hook is available to you: `const locale = useLocale()`
+- Within a page’s `getProps` you can call utility function `resolveLocaleFromUrl` like this:
+
+```js
+ProductDetail.getProps = async ({res, params, location, api}) => {
+    const locale = resolveLocaleFromUrl(`${location.pathname}${location.search}`)
+    ...
+}
```

Comment:
out of the scope of this PR...

it would be nice if we could make developer's life easier like this:

```suggestion
ProductDetail.getProps = async ({res, params, location, api, locale}) => {

}
```

@alexvuong